Dockerfile

```diff
@@ -12,7 +12,14 @@ LABEL maintainer=terraform-linters
 
 RUN apk add --no-cache ca-certificates
 
-COPY --from=builder /tflint/dist/tflint /usr/local/bin
+# copy tflint from builder
```

Comment:
Remove these comments, they are unnecessarily re-stating things that can be readily inferred from the code. Everything happening here is basic and well-documented upstream. You can assume basic understanding of Dockerfiles here and not try to explain how they work.